tasks/hyrax-metadata-updates/index.js

```diff
@@ -308,11 +308,21 @@ const updateGranule = (config) => async (granule) => {
   addEtagsToFileObjects(granule, etags);
   // Read in the metadata file
   const metadataFile = granule.files.find(isCMRFile);
-  // If there is no metadata file, error out.
+  // If there is no metadata file.
   if (metadataFile === undefined) {
+    // Skip metadata check if skipMetaCheck set to true
```

Comment:
👍🏻  this seems reasonable, but it needs a unit test addition  (e.g. https://github.com/nasa/cumulus/blob/master/tasks/hyrax-metadata-updates/tests/test-integration-index.js)